package.json

```diff
@@ -8,8 +8,11 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "start": "nodemon server/app.js"
```

Comment:
Don't forget to include a script in your package.json to run your seed file.

Reply:
Thanks for the tip, I have added the script to my package.json!